linksight/api/tests.py

```diff
@@ -92,13 +94,30 @@ def _test_case(self, test_case):
                     assert expected_val in map(lambda r: r[field], result)
 
     columns = OrderedDict([
-        ('bgy', 'bgy'),
-        ('municity', 'municity'),
-        ('prov', 'prov')
+        ('bgy', 'input_bgy'),
+        ('municity', 'input_mun'),
+        ('prov', 'input_prov')
     ])
 
-    def create_matcher(self, dataset, columns=''):
-        if not columns:
-            columns = self.columns
-
-        return NgramsMatcher(dataset, columns)
+    def create_matcher(self, dataset, columns=None):
+        return NgramsMatcher(dataset, columns or self.columns)
+
+    def test_clean_stats(self):
+        matcher = self.create_matcher(CLEAN_FILE)
+        (duration, accuracy) = profiling.get_stats(matcher)
```

Comment:
as discussed offline, probably better to make the this a dict instead of a tuple in case want to add more stats.